low-level/framework/platforms/realstor/realstor_enclosure.py

```diff
@@ -210,41 +212,61 @@ def ws_request(self, url, method, retry_count=MAX_RETRIES,
             post_data=""):
         """Make webservice requests using common utils"""
         response = None
-        relogin = False
+        retried_login = False
         tried_alt_ip = False
 
         while retry_count:
             response = self.ws.ws_request(method, url,
                        self.common_reqheaders, post_data,
                        self.WEBSERVICE_TIMEOUT)
 
+            #TMP
+            logger.error("CSDERR retry_count {0}".format(retry_count))
```

Comment:
Why multiple error statement. We should form a error string and call error at once.

Reply:
Have removed the error logs. It was for debug